### PR TITLE
Rust 1.59

### PR DESCRIPTION
**Pull request by davidiw**
Just having a little fun in the spirit of cleaning up and pondering documentation
* Remove framework-release since it is unused
* Upgrade to Rust-1.59 and see how badly things break

It may be worth just creating a massive stack of things to push out after the next branch cut cause 1.6 is coming on the 7th

### CODE REVIEW

File: aptos-move/transaction-replay/src/main.rs [hunk 1]
```diff
@@ -18,9 +18,6 @@ struct Opt {
     /// Path to the local AptosDB file
     #[structopt(long, parse(from_os_str))]
     db: Option<PathBuf>,
-    /// Full URL address to connect to - should include port number, if applicable
```
Comment (gregnazario) [inline aptos-move/transaction-replay/src/main.rs:21]: Lol why are you updating this?

File: storage/aptosdb/src/lib.rs [hunk 2]
```diff
@@ -134,7 +133,7 @@ static ROCKSDB_PROPERTY_MAP: Lazy<HashMap<&str, String>> = Lazy::new(|| {
         "rocksdb.block-cache-pinned-usage",
     ]
     .iter()
-    .map(|x| (*x, format!("aptos_{}", x.replace(".", "_"))))
+    .map(|x| (*x, format!("aptos_{}", x.replace('.', "_"))))
```
Comment (gregnazario) [inline storage/aptosdb/src/lib.rs:136]: Why not both to single quotes?  Or is that not supported?

**Review by gregnazario — APPROVED** [review]

File: storage/aptosdb/src/lib.rs (hunk 2 above)
Comment (davidiw) [reply storage/aptosdb/src/lib.rs:136]: the second one expects a string literal or something... rust is amazing

File: aptos-move/transaction-replay/src/main.rs (hunk 1 above)
Comment (davidiw) [reply aptos-move/transaction-replay/src/main.rs:21]: unused, apparently the dead code detection is much better

**Comment by davidiw** [comment]
hahaha, fun fighting with cargo xfmt ... 

/land

File: storage/aptosdb/src/lib.rs [hunk 3]
```diff
@@ -258,7 +208,6 @@ impl AptosDB {
             state_store: Arc::new(StateStore::new(Arc::clone(&db))),
             transaction_store: Arc::clone(&transaction_store),
             system_store: Arc::clone(&system_store),
-            rocksdb_property_reporter: RocksdbPropertyReporter::new(Arc::clone(&db)),
```
Comment (msmouse) [inline storage/aptosdb/src/lib.rs:261]: ??? why?

**Pull request merged**